HotSpot/View Controllers /ReportHomeownerViewController.m

```diff
@@ -0,0 +1,58 @@
+//
+//  ReportHomeownerViewController.m
+//  HotSpot
+//
+//  Created by aaronm17 on 8/6/19.
+//  Copyright © 2019 aodemuyi. All rights reserved.
+//
+
+#import "ReportHomeownerViewController.h"
+#import "RegexHelper.h"
+#import "EmailHelper.h"
+
+@interface ReportHomeownerViewController ()
+
+@property (weak, nonatomic) IBOutlet UITextView *reportField;
+
+@end
+
+@implementation ReportHomeownerViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+}
+
+- (IBAction)didTapCancel:(UIBarButtonItem *)sender {
+    [self dismissViewControllerAnimated:YES completion:nil];
+}
+
+- (IBAction)didTapView:(UITapGestureRecognizer *)sender {
+    [self.view endEditing:(YES)];
+}
+
+//checks if the report field is empty, if not sends a specialized report
+- (IBAction)didTapSendReport:(UIButton *)sender {
+    if ([RegexHelper isEmpty:self.reportField.text]) {
+        UIAlertController *alert = [RegexHelper createAlertController];
+        
+        alert.title = @"Empty text field";
```

Comment:
this seems to be a common flow - create alert with message and title and present it. your create alert helper should take in a message and text